docs/compiling-mono/mac/index.md

```diff
@@ -43,6 +43,12 @@ for i in *.tar.gz; do tar xzvf $i; done
 for i in */configure; do (cd `dirname $i`; ./configure --prefix=$PREFIX && make && make install); done
 ```
 
+Alternatively, dependencies can be installed through Homebrew:
+
+``` bash
+brew install autoconf automake libtool pkg-config glib libmcrypt freetype cmake libmemcached
```

Comment:
Actually, you can get the dependencies list with `brew deps mono`. I think it would be something like `brew install $(brew deps mono)`